client/src/router/AppRoutes.tsx

```diff
@@ -0,0 +1,48 @@
+import React, {JSX} from 'react';
+import LandingPage from '../component/page/LandingPage';
+import FormPage from '../component/page/FormPage';
+import ErrorPage from '../component/page/ErrorPage';
+import { ERROR_MESSAGES } from '../component/error/ErrorMessages';
+import { Navigate } from 'react-router-dom';
+
+type RouteType = {
+  path: string,
+  element: JSX.Element
+}
+
+type RoutePaths = {
+  [key: string]: string;
+}
+
+const ROUTE_PATHS: RoutePaths = {
+  ROOT: '/',
+  FORM: '/form',
+  ERROR: '/error',
+  OTHER: '*'
+};
+
+const ROUTES: RouteType[] = [
```

Comment:
This defines all possible routes that a user can navigate to. At the moment if a user navigates to:

- `/` they see the landing page
- `/form` they see the form page
- `/error` they see the default error page with the default error message
- `*` this is the wild card for any other route, what happens is if a user goes to an invalid url they are re-directed to `/error` and they get the  page not found message on the error page